arbnode/forwarder.go

```diff
@@ -149,6 +167,7 @@ func (f *TxForwarder) Initialize(inctx context.Context) error {
 // Disable is not thread-safe vs. Initialize
 func (f *TxForwarder) Disable() {
 	atomic.StoreInt32(&f.enabled, 0)
+	// TODO should we close ethClient / rpcClient here or in StopAndWait?
```

Comment:
Let's close it in StopAndWait. Disable should let any currently in-flight requests go through.